share/spice/holiday/holiday.js

```diff
@@ -68,7 +76,7 @@
                 // specified let that trump tense.
                 var event_date = moment(new Date(data.o[0].d));
                 var current_date = moment();
-                if ((tense === "was") && event_date.isAfter(current_date) && (year.length == 0)) {
+                if ((tense === "was") && event_date.isAfter(current_date) && (year.length === 0)) {
```

Comment:
Minor comment, the additional brackets here don't add anything except to make it harder to read, I'd consider removing them.